src/utils/get-fastest-provider.ts

```diff
@@ -0,0 +1,26 @@
+// @ts-expect-error - no types
```

Comment:
`rpc-handler` seems to have types?

Reply:
The repo does but the package does not for some reason. My test pkg I published has types and it's the same flow that was used to publish but the UBQ pkg doesn't have types in either of the versions on npmjs

https://www.npmjs.com/package/@ubiquity-dao/rpc-handler?activeTab=code